client/me/account-password/index.jsx

```diff
@@ -90,7 +92,7 @@ module.exports = React.createClass( {
 					debug( 'Error saving password: ' + JSON.stringify( error ) );
 
 					// handle error case here
-					notices.error( this.translate( 'There was a problem saving your password. Please, try again.' ) );
+					this.props.errorNotice( this.translate( 'There was a problem saving your password. Please, try again.' ) );
```

Comment:
While I agree with trying to make it more readable, because of sheer number of files to convert in these notice-reduxification efforts, I am trying to keep changes as minimal as possible.

Also, I am not a big fan of destructuring - I like to see instantly where the function came from :)